sqliter-driver/src/linuxX64Main/kotlin/co/touchlab/sqliter/internal/File.kt

```diff
@@ -431,7 +431,7 @@ internal actual class File(dirPath: String? = null, name: String) {
     }
 
     private fun mkdirImpl(filePath: String): Boolean {
-        return mkdir(filePath, S_IRWXU) == F_OK
+        return mkdir(filePath, S_IRWXU.toUInt()) == F_OK
```

Comment:
`S_IRWXU` is an `Int` but `mkdir` signature is now `@kotlinx.cinterop.internal.CCall public external fun mkdir(@kotlinx.cinterop.internal.CCall.CString __path: kotlin.String?, __mode: platform.posix.__mode_t /* = kotlin.UInt */): kotlin.Int { /* compiled code */ }`